app/components/Header.jsx

```diff
@@ -1,10 +1,10 @@
-import React, { PureComponent, PropTypes } from 'react';
+import React, { Component, PropTypes } from 'react';
 import { Link } from 'react-router';
 import Logo from 'svg-react-loader?name=Logo!../assets/images/logo.svg'; // eslint-disable-line
 
 import Search from './Search';
 
-export default class Header extends PureComponent {
+export default class Header extends Component {
```

Comment:
This should be `PureComponent` still... switched it when I was debugging and forgot to switch back.